### DISCUSSION TITLE
How can I do route like https://spooky.fi/#/pools ?

### DISCUSSION THREAD

**Discussion by dominixz-dzvz** [nuxt/nuxt#17338] [Questions]
I want to do app that can deploy to IPFS but it has a problem that when I "yarn generate" and deploy it. It has a problem that when I click from email to the path it failed something like "localhost:8080/myprofile" but when I start with "localhost:8080" and then click to my profile (url also the same as localhost:8080/myprofile" it show up

So I think it a problem is that it doesn't enter index.html when enter with "/myprofile" and i see https://spooky.fi/#/pools using # to parse url and then it will go to that page when I enter URL from browser.

How can I do the same with Nuxt?

**Comment by Flanker72** [comment]
You may try to have a look at [Universal router](https://v3.nuxtjs.org/api/composables/use-router#universal-router-instance).
Seems you can define custom history mode and set your own routing model like this:
```ts
<script setup lang="ts">
import { createWebHashHistory } from 'vue-router';
const { options } = useRouter();
options.history = createWebHashHistory();
...
</script>
```

**Reply by dominixz-dzvz** [reply]
So I have to giveup benefit of "pages" folder right ?

**Reply by Flanker72** [reply]
Looks like this. As written in the [official docs](https://v3.nuxtjs.org/guide/directory-structure/pages#router-options), **"Note: history and routes options will be always overridden by Nuxt."**.
If you need your own approach, you definitely need to implement custom routing.
Code reference: https://github.com/nuxt/framework/blob/f6bf4f75591e523aabc5ad98952a7118ddf77ca5/packages/nuxt/src/pages/runtime/router.ts#L59

**Reply by dominixz-dzvz** [reply]
Thank you so much. Now I just change that file in node_modules in my own project from createWebHistory to createWebHashHistory and then i tried yarn generate it work like magic. Wait for official to come.
